docs/fsharp/index.md

```diff
@@ -1,10 +1,9 @@
 ---
 title: F# Guide
-description: Learn about F#, a functional programming language that runs on .NET.
-keywords: .NET, .NET Core
+description: Learn F#, a functional programming language that runs on .NET.
```

Comment:
Try to expand this a bit more. The length should be between 115 and 145 characters. Maybe tell folks what they'll find in the Guide?